process/interceptors/processor/shardValidatorInfoInterceptorProcessor_test.go

```diff
@@ -118,7 +118,7 @@ func Test_shardValidatorInfoInterceptorProcessor_Save(t *testing.T) {
 	})
 }
 
-func Test_shardValidatorInfoInterceptorProcessor_DisabledMethods(t *testing.T) {
+func Test_validatorInfoInterceptorProcessor_DisabledMethods(t *testing.T) {
```

Comment:
TestValidatorInfoInterceptorProcessor_DisabledMethod ? 
This suggestion will comply with the rest of the project's naming conventions